tests/__snapshots__/test_voice.ambr

```diff
@@ -17,10 +18,28 @@
     'speak': dict({
       '@version': '1.0',
       '@xml:lang': 'nl-NL',
+      '@xmlns': 'http://www.w3.org/2001/10/synthesis',
+      '@xmlns:mstts': 'https://www.w3.org/2001/mstts',
       'voice': dict({
         '#text': 'Text for Saying',
-        '@name': 'Microsoft Server Speech Text to Speech Voice (nl-NL, FennaNeural)',
-        '@xml:lang': 'nl-NL',
```

Comment:
Are these 2 changes expected?

Reply:
Yeah, it was not needed. Looks like we were using an old SSML format.